### PR TITLE
Remove old ts-generator

### PR DESCRIPTION
**Pull request by alfonsograziano**
### Ticket:

- [15782](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/15782)

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [ ] **New feature**
- [X] **Dev change**

### CODE REVIEW

File: apps/payments/app/[locale]/(hosted)/paymentSetup/providers/[providerId]/EditProviderForm.tsx [hunk 1]
```diff
@@ -17,19 +16,16 @@ export default async ({
 }: PropsWithChildren<Props>) => {
   const t = await getTranslations("PaymentSetup.Providers.edit");
 
-  async function setProviderStatus(status: string) {
+  async function setProviderStatus(status: "connected" | "disconnected") {
```
Comment (peschina) [inline apps/payments/app/[locale]/(hosted)/paymentSetup/providers/[providerId]/EditProviderForm.tsx:19]: Can we reference the `ProviderStatus` type here?

File: apps/payments/app/integration/stripe.ts [hunk 2]
```diff
@@ -11,12 +11,10 @@ const getSecretKey = async (
...
 ): Promise<string> => {
-  const provider = (
-    await buildApiClient(userId).providers.apiV1ProvidersProviderIdGet(
-      providerId,
-    )
-  ).data;
-  return provider.data.liveSecretKey;
+  const provider = (await new Payments(userId).getProviderById(providerId))
+    .data;
+  const providerData: any = provider?.data;
```
Comment (peschina) [inline apps/payments/app/integration/stripe.ts:16]: can we use `StripeData` type here?

**Review by peschina — APPROVED** [review]
LGTM, a few minor comments

File: apps/payments/app/[locale]/(hosted)/paymentSetup/providers/[providerId]/EditProviderForm.tsx (hunk 1 above)
Comment (alfonsograziano) [reply apps/payments/app/[locale]/(hosted)/paymentSetup/providers/[providerId]/EditProviderForm.tsx:19]: Good catch!

**Pull request merged**